src/core_modules/capture-core/metaDataStoreLoaders/programs/quickStoreOperations/storePrograms.js

```diff
@@ -24,6 +24,8 @@ const convert = (() => {
         (apiProgramSections || [])
             .map(apiProgramSection => ({
                 ...apiProgramSection,
+                description: undefined,
+                displayDescription: apiProgramSection.description || '',
```

Comment:
@eirikhaugstulen 
Looks like `displayDescription` is returned by the [API](https://play.im.dhis2.org/dev/api/42/programs?fields=id,programSections[id,displayDescription,description]) too. Can it be used instead of `description`? 
Thanks!

Reply:
You live and you learn! Thanks!